File: LeetCode/Simulation/Golang/Leetcode1051-heightChecker.go
package sbgo

import "sort"

func heightChecker(heights []int) (ans int) {
	n := len(heights)
	rec := make([]int, n)
	for i := range heights {
		rec[i] = heights[i]
	}
	sort.Ints(rec)
	for i := 0; i < n; i++ {
		if rec[i] != heights[i] {
			ans++
		}
	}
	return
}

File: LeetCode/Simulation/JS/Leetcode1051-heightChecker.js
var heightChecker = function (heights) {
    let [rec, n] = [[...heights], heights.length];
    let ans = 0;
    rec.sort((a, b) => a - b);
    for (let i = 0; i < n; i++) {
        if (rec[i] != heights[i]) ans++;
    }
    return ans;
};